samples/lcd/lcd-alphabet/main.cpp

```diff
@@ -0,0 +1,53 @@
+//
+// Created by Zachary Lineman on 11/14/22.
+//
```

Comment:
Assuming this is CLion autogenerated. Don't think this follows our style guidelines